libs/wingc/src/jsify.rs

```diff
@@ -251,12 +251,19 @@ impl<'a> JSifier<'a> {
 	}
 
 	fn jsify_user_defined_type(&self, user_defined_type: &UserDefinedType) -> String {
+		// "std" is implicitly imported, so replace with "$stdlib.std".
+		let root = if user_defined_type.root == WINGSDK_STD_MODULE.into() {
+			format!("{STDLIB}.{WINGSDK_STD_MODULE}")
+		} else {
+			user_defined_type.root.to_string()
+		};
```

Comment:
Is there a way to avoid adding this edge case? I think it might be possible to remove the `$stdlib` identifier we made up, and use `std` directly

Reply:
I kind of like this solution. Let me look into it.